src/test/java/io/github/sskorol/converters/StringToBooleanConverter.java

```diff
@@ -0,0 +1,10 @@
+package io.github.sskorol.converters;
+
+import com.creditdatamw.zerocell.converter.Converter;
+
+public class StringToBooleanConverter implements Converter<Boolean> {
```

Comment:
There's an existing [BooleanConverter](https://github.com/creditdatamw/zerocell/blob/master/core/src/main/java/com/creditdatamw/zerocell/converter/BooleanConverter.java#L5) in ZeroCell.